### PR TITLE
Prepare links for weekend (day 11 and 12)

### PR DESCRIPTION
**Pull request by vincenzobaz**
Prepare links for solutions contributed by the community over the weekend

### CODE REVIEW

File: website/src/pages/index.js [hunk 1]
```diff
@@ -66,10 +66,20 @@ function HomepageHeader() {
...
             Day 10
           </Link>
+          <Link
+              className={`${styles.button} button button--secondary button--lg`}
+              to="/puzzles/day11">
+            Day 11
+          </Link>
+          <Link
+              className={`${styles.button} button button--secondary button--lg`}
+              to="/puzzles/day12">
```
Comment (mlachkar) [inline website/src/pages/index.js:80]: I think it can be nice as disabled ..

**Comment by mlachkar** [comment]
not sure it wont fail when deploying the website, since the pages are not created yet :s

**Comment by mlachkar** [comment]
add empty pages maybe, like in this commit https://github.com/scalacenter/scala-advent-of-code/pull/32/files

**Comment by vincenzobaz** [comment]
> not sure it wont fail when deploying the website, since the pages are not created yet :s

[Here](https://github.com/scalacenter/scala-advent-of-code/tree/website/docs/puzzles) I see that all pages are there. Is this what you are referring to?

**Review by mlachkar — APPROVED** [review]

**Comment by mlachkar** [comment]
> > not sure it wont fail when deploying the website, since the pages are not created yet :s
> 
> [Here](https://github.com/scalacenter/scala-advent-of-code/tree/website/docs/puzzles) I see that all pages are there. Is this what you are referring to?

I didn't notice! thanks

**Pull request merged**